subsys/net/l2/ieee802154/ieee802154_frame.c

```diff
@@ -916,19 +936,24 @@ bool ieee802154_create_ack_frame(struct net_if *iface, struct net_pkt *pkt, uint
 bool ieee802154_decipher_data_frame(struct net_if *iface, struct net_pkt *pkt,
 				    struct ieee802154_mpdu *mpdu)
 {
+	bool ret = false;
```

Comment:
try to keep a reversed christmas tree variable ordering when applicable.

Reply:
Fixed, but where can I find this rule (and others you claimed) documented? I looked in the [coding guidelines](https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html) and my code is passing all automatic style checks. If such rules are expected by the project why are they not enforced by the style checker and/or documented? Or did I simply overlook them?